src/Management/test/Endpoint.Test/Actuators/Health/HealthEndpointTest.cs

```diff
@@ -178,16 +178,21 @@ public async Task InvokeWithLivenessGroupReturnsGroupResults()
 
         testContext.AdditionalConfiguration = configuration => configuration.AddInMemoryCollection(new Dictionary<string, string?>
         {
-            ["Management:Endpoints:Health:ShowDetails"] = "Always"
+            ["Management:Endpoints:Health:ShowComponents"] = "Always",
+            ["Management:Endpoints:Health:Liveness:Enabled"] = "true"
         });
 
         var appAvailability = new ApplicationAvailability(NullLogger<ApplicationAvailability>.Instance);
-        TestOptionsMonitor<LivenessHealthContributorOptions> optionsMonitor = new();
+
+        var optionsMonitor = TestOptionsMonitor.Create(new LivenessStateContributorOptions
+        {
+            Enabled = true
+        });
```

Comment:
Let's avoid having duplicated configuration settings in different shapes. Instead, extract the `appSettings` dictionary into a variable and use:
```c#
var optionsMonitor = GetOptionsMonitorFromSettings<LivenessStateContributorOptions>(appSettings);
```